src/app/screens/createInscription/index.tsx

```diff
@@ -254,6 +254,8 @@ function CreateInscription() {
     seedPhrase,
     contentBase64: payloadType === 'BASE_64' ? content : undefined,
     contentString: payloadType === 'PLAIN_TEXT' ? content : undefined,
+    serviceFee: appFee,
```

Comment:
I was wondering about this too. When I debugged the extension, I questioned why you don't call it either `serviceFee` or `appFee` in both the extension and sats-connect.